hosting.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Hosting
+layout: recommendation
+expires: 2017-11-01
+---
+
+GDS follows the [Government Cloud First policy](https://www.gov.uk/guidance/government-cloud-first-policy) meaning that we prefer the use of PaaS and IaaS solutions over managing our own hardware
+
+## User Needs
+
+* Users want systems that are highly scalable and available
+* GDS wants to manage the environments easily and simply, with confidence via automated tools
+
+## Principles
+
+GDS will build new tools on only one cloud platform for each of IaaS and PaaS.
```

Comment:
I was wondering what the scope of "new tools" is? As an example, if we wanted to use storage services for something like offsite backups or DNS services I think it would make sense to allow the scope to use more than a single vendor.